react-native-libraries.json

```
@@ -17082,5 +17082,28 @@
     "android": true,
     "expoGo": true,
     "newArchitecture": true
+  },
+  {
+    "githubUrl": "https://github.com/NIKHIL0VERMA/flixsrota-player",
+    "npmPkg": "@flixsrota/player",
+    "examples": [
+      "https://github.com/NIKHIL0VERMA/flixsrota-player/tree/main/example"
+    ],
+    "images": [
+      "https://raw.githubusercontent.com/NIKHIL0VERMA/flixsrota-player/main/assets/buffering.jpg",
+      "https://raw.githubusercontent.com/NIKHIL0VERMA/flixsrota-player/main/assets/controls.jpg",
+      "https://raw.githubusercontent.com/NIKHIL0VERMA/flixsrota-player/main/assets/fullscreen_player.jpg",
+      "https://raw.githubusercontent.com/NIKHIL0VERMA/flixsrota-player/main/assets/skip_gesture.jpg"
+    ],
+    "ios": true,
+    "android": true,
+    "web": false,
+    "expoGo": true,
+    "unmaintained": false,
+    "dev": false,
+    "template": false,
+    "alternatives":[
+      "react-native-youtube-iframe"
+    ]
```

Hey @NIKHIL0VERMA, thanks for adding a new entry to the directory! 👍 

Just two small suggestion before we can ship it:
* `alternatives` list only applies to libraries which are not maintained anymore, or do not support New Architecture, `react-native-youtube-iframe` seems to do not qualify for those criterions, more details:
  * https://github.com/react-native-community/directory?tab=readme-ov-file#alternatives
* flags with `false` values can be safely skipped

```suggestion
    "expoGo": true
```